fasttrips/PathSet.py

```diff
@@ -83,6 +83,15 @@ class PathSet:
     #: into A-B-C-D-E for overlap calculations?
     OVERLAP_SPLIT_TRANSIT           = None
 
+    #: Allow departures and arrivals before / after preferred time
+    PAT_VARIANCE                   = 0
+
+    #: Weight applied to each minute before or after preferred time
+    PAT_PENALTY                    = 1.01
+
+    #: Does the penalty grow exponentially
+    PAT_PENALTY_EXP                = True
```

Comment:
The initial take on this is in there now with e07bb23. 

I had to refresh my calculus a little on the exponential since we are looking for the area under the curve of (1+growth rate)**minutes. I crossed reference my formula across a few different websites, and I am pretty sure its right. A second set of eyes would be good though.

https://github.com/danielsclint/fast-trips/blob/e07bb23d8f16ae6ceb5a9080cc607c41fd0639d8/fasttrips/PathSet.py#L971